partner_survey/wizard/take_survey.py

```diff
@@ -0,0 +1,28 @@
+from odoo import _, fields, models
+
+
+class take_survey_wizard(models.Model):
+    """Wizard allowing to take a survey from partners form view"""
+
+    _name = "res.partner.survey.survey.wizard"
+
+    partner_ids = fields.Many2many("res.partner")
```

Comment:
Should this be editable?

```suggestion
    partner_ids = fields.Many2many("res.partner", required=True)
```

Reply:
What about this?